blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/horizontal-links/default.jsx

```diff
@@ -1,23 +1,15 @@
 import React from "react";
-import styled from "styled-components";
 import { useContent } from "fusion:content";
 import { useFusionContext } from "fusion:context";
 import getProperties from "fusion:properties";
 import getTranslatedPhrases from "fusion:intl";
-import { PrimaryFont } from "@wpmedia/shared-styles";
-import Link from "./_children/link";
+import { Stack, Link, Separator } from "@wpmedia/arc-themes-components";
 
-import "./links-bar.scss";
-
-const ReadableTextNavigationBar = styled.nav`
-	color: ${(props) => props.color};
-
-	a {
-		color: ${(props) => props.color};
-	}
-`;
-
-const HorizontalLinksBar = ({ hierarchy, navBarColor, showHorizontalSeperatorDots, ariaLabel }) => {
+const HorizontalLinksBar = ({
+	hierarchy,
+	showHorizontalSeperatorDots: showHorizontalSeparatorDots,
```

Comment:
Are they always horizontal?  Does horizontal carry significance here?

Reply:
This has been changed to `NavLinksBar`

Reply:
This was changed to `NavLinksBar`